source/content/guides/multisite/03-config.md

```diff
@@ -68,20 +68,43 @@ Make sure [Terminus](/terminus) is installed and [authenticated](/terminus/insta
 
 1. Open the `code` folder in your SFTP client, and download your site's `wp-config.php` file.
 
-1. Locate the configuration added by WP-CLI, and *modify* the line that sets `DOMAIN_CURRENT_SITE` from a hardcoded URL to a dynamic URL `$_SERVER['HTTP_HOST']`. This automatically detects the URL in each environment. You must replace this variable. For example:
-
+1. Locate the configuration added by WP-CLI, and *modify* the line that sets `DOMAIN_CURRENT_SITE` to a hardcoded URL. You will want to default to a dynamic URL (`$_SERVER['HTTP_HOST']`), however some environments on Pantheon (notably workflows that run containerized versions of WP-CLI like search and replace) will not have a `$_SERVER` global. For this reason, we recommend using the following configuration that provides fallbacks if `$_SERVER['HTTP_HOST']` is unavailable.
+  
   ```php:title=wp-config.php
-  define( 'WP_ALLOW_MULTISITE', true );
+  $hostname = 'www.yourdomain.com'; // The domain of the network. Use as a fallback if $_SERVER['HTTP_HOST'] is not available.
+  if ( !empty( $_ENV['PANTHEON_ENVIRONMENT'] )) {
+    $site_name = $_ENV['PANTHEON_SITE_NAME'];
```

Comment:
In PR about checking that array keys exist before using them, check this key too :)

Reply:
If we are on a `PANTHEON_ENVIRONMENT` (e.g. `$_ENV['PANTHEON_ENVIRONMENT']` is not empty), we can make a lot of assumptions. One of those assumptions is that `PANTHEON_SITE_NAME` would _also_ exist. The issue is not the `isset` itself, the issue is that in some environments (predominantly unified job runner workflows that do not actually spin up a live server) the `$_SERVER` superglobal is not defined. If `$_ENV['PANTHEON_ENVIRONMENT']` is defined, we don't need to worry about that.